draft-ietf-anima-brski-prm.md

```diff
@@ -336,84 +320,84 @@ Solution examples based on existing technology are provided with the focus on ex
 # Architectural Overview {#architecture}
 
 For BRSKI with pledge in responder mode, the base system architecture defined in BRSKI {{RFC8995}} is enhanced to facilitate the new use cases in which the pledge acts as server.
-The pledge-responder-mode allows delegated bootstrapping using a registrar-agent instead of a direct connection between the pledge and the domain registrar.
+The responder mode allows delegated bootstrapping using a registrar-agent instead of a direct connection between the pledge and the domain registrar.
 
 Necessary enhancements to support authenticated self-contained objects for certificate enrollment are kept at a minimum to enable reuse of already defined architecture elements and interactions.
 
 For the authenticated self-contained objects used for the certification request, BRSKI-PRM relies on the defined message wrapping mechanisms of the enrollment protocols stated in {{req-sol}} above.
 
-The security used within the document for bootstrapping objects produced or consumed by the pledge bases on JOSE {{RFC7515}}. In constrained environments it may be provided based on COSE {{RFC9052}} and {{RFC9053}}.
+The security used within the document for bootstrapping objects produced or consumed by the pledge bases on JOSE {{RFC7515}}.
+In constrained environments it may be provided based on COSE {{RFC9052}} and {{RFC9053}}.
 
 An abstract overview of the BRSKI-PRM protocol can be found in  {{BRSKI-PRM-abstract}}.
```

Comment:
Cannot remain a GitHub resource. Can HTML RFCs now have complex figures in the appendix? Otherwise move it outside the draft, e.g., to brski.org?

Reply:
Also exchanges come again in Section 6. Overlap?

Reply:
The figure was used in the WG meetings to provide an overview of BRSKI-PRM and Sheng proposed to include the figure here. It is a more detailed view on the call flows and the architecture as provided in section 6. If there is a better place for an informal reference that the git, we may change it. Will ask Micheal regarding brski.org

Reply:
Sure it is helpful. I am just not sure about references to unstable resources from a long-lived document. But more and more RFCs appear to have references to blogs or news paper articles.